src/components/Text/Text.module.css

```diff
@@ -5,8 +5,78 @@
 \*------------------------------------*/
 
 /**
- * 1) 
+ * 1) h1, h2, h3, h4, h5, h6
+ * 2) Styles are either controlled by as or size declaration
  */
-/* stylelint-disable-next-line block-no-empty */
-.text {
+.text,
+.text--body, 
+.text--md {
+  @mixin reset;
+  
+  @mixin eds-theme-typography-body-text;
+}
+
+/**
+ * Sizes
+ */
+.text--lg {
+  @mixin eds-theme-typography-body-text-lg;
+}
+.text--sm {
+  @mixin eds-theme-typography-body-text-sm;
+}
+.text--xs {
+  @mixin eds-theme-typography-body-text-xs;
+}
+.text--caption {
+  @mixin eds-theme-typography-caption-text-md;
+}
+.text--overline {
+  @mixin eds-theme-typography-kicker;
+}
+
+/**
+ * Color
```

Comment:
`base`, `brand`, `success`, `neutral` colors differ from v0, but I'm assuming that's intended via design